samples/cellular/modem_shell/sample.yaml

```diff
@@ -238,7 +238,7 @@ tests:
     integration_platforms:
```

Comment:
Cannot add a comment on line 226 which has `sample.cellular.modem_shell.memfault_modem_trace.nrf9161` build. Should this be duplicated for 9151, or renamed as 91x1 and 9151 added into `platform_allow`?

Same comment for `sample.cellular.modem_shell.modem_fota_full.nrf9161` on line 111.